src/java/com/wolfssl/WolfSSLCustomUser.java

```diff
@@ -0,0 +1,62 @@
+
+/* WolfSSLCustomUser.java
+ *
+ * Copyright (C) 2006-2019 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+package com.wolfssl;
+/**
+ * Base class is intended to give some customizing points. Currently it is limited
+ * to be invoked from WolfSSLContext.Create
+ *
+ * @author  wolfSSL
+ * @version 1.0, August 2013
+ */
+public class WolfSSLCustomUser {
+    public long method = 0;
+    public String[] list;
+
+    /**
+     * Set callback for Context attributes, TLS protocol and Cipher list
+     *
+     *      WARNING: inappropriate code or use of this callback may cause serious security issue.
+     *
+     * @param method       default version of TLS method for refernce.
+     * @param list         default cipher list for refernce.
+     * @return             method: Lowest TLS protocol version allowed to the context. The value has
```

Comment:
The method returned will be used with wolfSSL_CTX_new().  It should not be the "lowest" protocol version allowed - it should be the "highest" version allowed.

Reply:
Fixed the comment